### ISSUE TITLE
Requesting for the checkpoint for the model trained with the REDS dataset

### ISSUE THREAD

**Issue by codeslake** [zzh-tech/ESTRNN#21]
Hi, 

Will it be possible for you to share the checkpoint of the model trained with the REDS dataset?

By the way, just to make sure, the score reported in the main paper evaluated with the validation set of the REDS dataset, right?
As far as know, the authors of the REDS dataset don't share ground truths for the test set.

**Comment by zzh-tech** [comment]
> Will it be possible for you to share the checkpoint of the model trained with the REDS dataset?

Hi, this work was done during my internship at Huawei.
I don't have permission to copy any checkpoints from the server.

> By the way, just to make sure, the score reported in the main paper evaluated with the validation set of the REDS dataset, right?

Yes.

**Comment by codeslake** [comment]
I see, thanks for the reply!
